### ISSUE TITLE
[Feature] Tag support/mod blacklist/wildcard and an ore highlighter command

### ISSUE THREAD

**Issue by Nevrai** [EwyBoy/World-Stripper#25]
### Suggestion 1: tag support and/or a mod blacklist or wildcard

Adding all the ores you want to strip is a time-consuming task, so it would be really nice if you could add tags or if you could add the name of a mod or use wildcards (e.g., `/worldstripper add thermal:*_ore`).

### Suggestion 2: ore highlighter

I think it would be incredibly useful for ore generation testing if you could type a command that highlights all nearby blocks of that type. E.g., type `/worldstripper highlight minecraft:diamond_ore` and it adds a glowing outline around all diamond ores in your viscinity. That would be fantastic!

**Comment by EwyBoy** [comment]
Thanks!
Both really cool suggestions. I will look into it once I have the time and keep you posted 👍

**Comment by nikita488** [comment]
Yeah, will be cool to tags and whitelist/blacklist support, because right now if player wants to debug only specific Mod worldgen, you need to add all other worldgen blocks to the strip profile.
